.gitignore

```diff
@@ -44,6 +44,7 @@ src/couch/priv/couch_js/util.d
 src/couch/priv/icu_driver/couch_icu_driver.d
 src/mango/src/mango_cursor_text.nocompile
 src/docs/
+src/emilio/
```

Comment:
Add `bin/emilio` too

Reply:
bin/ directory should be ignored already